Update application-model.md (#1016)
MicrosoftDocs/entra-docs: docs/identity-platform/application-model.md

## Multitenant apps

> [!IMPORTANT]
>

> > Multi-tenant applications (MTAs) do not function across cloud boundaries due to the segregation of service principal authorities within each cloud. For instance, if the application object is hosted in the commercial cloud, the associated service principal is created locally during customer onboarding. This process fails when crossing cloud boundaries because the authority URLs differ (e.g., `.com` vs. `.us`), causing an incompatibility.

Reviewer: ```suggestion
> Multitenant applications (MTAs) don't function across cloud boundaries due to the segregation of service principal authorities within each cloud. For instance, if the application object is hosted in the commercial cloud, the associated service principal is created locally during customer onboarding. This process fails when crossing cloud boundaries because the authority URLs differ (e.g., `.com` vs. `.us`), causing an incompatibility.
```
fixing Microsoft style issues
